street_network/src/render.rs

```diff
@@ -49,7 +50,15 @@ impl StreetNetwork {
                     ("osm_node_id", id.0.into()),
                     (
                         "complexity",
-                        format!("{:?}", intersection.complexity).into(),
+                        if intersection.complexity == MultiConnection {
```

Comment:
Mostly, but so far `Terminus | Connection => Uncontested` and `Crossing => Cross`, so this keeps the changes to the test output relevant for now. (I have not committed the tests, which are failing, so the diff will remain enlightened as I go.)

A new JSON field is likely the best option in the end.

Reply:
Another option is being more precise with the possibilities..

```
enum Complexity {
  Terminus,
  Connection,
  Crossing(ConflictType),
}
```

since the conflict is unambiguous for some cases. But then if some piece of code needs to reason about the special cases, this might be more awkward

Reply:
I had both those thoughts too. I think always having it around, and probably also always calculating it, is simpler. Our assumptions are bound to be wrong in some ways, too.